deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Maybe make this bin/bash just to be safe. I remember that bash has features that sh doesn't and it could prevent bugs in the future to use the superset of sh, bash.

Reply:
Most of the sample scripts I saw online seemed to use sh rather than bash - not sure why but I'll check on that. I was following those for now, but I'll check on this